drivers/i2s/Kconfig.nrfx

```
@@ -22,3 +22,24 @@ config I2S_NRFX_TX_BLOCK_COUNT
 	default 4
 
 endif # I2S_NRFX
+
+menuconfig TDM_NRF
+	bool "nRF TDM nrfx driver"
```

```suggestion
menuconfig I2S_NRF_TDM
	bool "nRF TDM driver"
```
also:
- `TDM_NRFX_*X_BLOCK_COUNT` -> `I2S_NRF_TDM_*X_BLOCK_COUNT`
- `i2s_nrfx_tdm.c` -> `i2s_nrf_tdm.c`